csharp/src/Ice/Internal/PropertyNames.cs

```diff
@@ -457,13 +457,6 @@ public sealed class PropertyNames
         "IceDiscovery",
         "IceLocatorDiscovery",
         "IceBox",
-        "IceBoxAdmin",
```

Comment:
I would keep these reserved prefixes. They are reserved in all languages, even though we don't use them in all languages.